pkg/build/util/util.go

```diff
@@ -9,8 +9,12 @@ import (
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
-// BuildPodSuffix is the suffix used to append to a build pod name given a build name
-const BuildPodSuffix = "build"
+const (
+	// BuildPodSuffix is the suffix used to append to a build pod name given a build name
+	BuildPodSuffix = "build"
+	// NoBuildLogsMessage reports that no build logs are available
+	NoBuildLogsMessage = "No logs are available"
```

Comment:
period at the end, to align it with the rest of the message you're appending this to.